docs/source/conf.py

```diff
@@ -109,3 +111,6 @@
 
 # Output file base name for HTML help builder.
 # htmlhelp_basename = 'doc'
+
+# Exposes our custom module_utils content for docstring
+sys.path.append("../../ci_framework/plugins/module_utils")
```

Comment:
@pablintino maybe we can tweak something here so that it loads with the same ansible_collections namespace or something?